### DISCUSSION TITLE
How to handle remixed art.

### DISCUSSION THREAD

**Discussion by TheGuyWhoo** [DISTRHO/Cardinal#284] [General]
Do I just clone the repository, push the changes and then add that repo as a submodule or what?

I have it building and dark versions of the the panels.

![Screenshot from 2022-07-06 00-41-41](https://user-images.githubusercontent.com/72744967/177429062-4bde2ace-68cd-401f-a93e-b531f3ac82fc.png)

**Comment by falkTX** [comment]
The dark versions are typically done automatically, otherwise we would need to maintain a fork of the project.
With so many modules that exist, that is not a good choice to go for by default.
If an automatic filter works, it is preferred.
